app/containers/Emoji/Emoji.tsx

```
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Text, useWindowDimensions } from 'react-native';
+import type { StyleProp, TextStyle, ImageStyle } from 'react-native';
+
+import { useTheme } from '../../theme';
+import useShortnameToUnicode from '../../lib/hooks/useShortnameToUnicode';
+import CustomEmoji from '../EmojiPicker/CustomEmoji';
+import { useAppSelector } from '../../lib/hooks/useAppSelector';
```

ESLint

[error] 'useAppSelector' is defined but never used.

(@typescript-eslint/no-unused-vars)